CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # [vNext]
 
 ## Improvements:
+* Added sorting of the list of plugins which is given back during plugin discovery (#519)
```

Comment:
Could you please rephrase this to describe the change from the user's perspective? Something like "Ensure that the runtime plugins are loaded in an alphabetic order based on file name on Unix-based platforms."